test/integration/app/wallets/send-lightning.spec.ts

```diff
@@ -111,6 +111,59 @@ describe("lightningPay", () => {
       await Transaction.deleteMany({ memo })
       lndServiceSpy.mockReset()
     })
+
+    it("fails if sends to self", async () => {
+      const memo = randomLightningMemo()
+
+      // Setup mocks
+      const { LndService: LnServiceOrig } = jest.requireActual("@services/lnd")
```

Comment:
I like this usage of a mock. It's clearly scoped and not very complex to setup. Also easy to identify the relationship with what is being tested.

I'm fine using more like this for there new integration tests.